doc/vector/code-samples/zvkg.h

```diff
@@ -29,4 +29,31 @@ zvkg_vghsh(
     const void* H
 );
 
+// Y, X, and H point to arrays of 128 bits values, 32b aligned
+// if the processor does not support unaligned access.
+// 'n' is the number 128b element groups.
```

Comment:
is "ok" missing here ?
```suggestion
// 'n' is the number of 128b element groups.
```

Reply:
Indeed. Done